apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -53,24 +69,32 @@ describe('Button Functional Testing', () => {
   });
 
   it('Validate OnClick() callback was fired -> Click', async () => {
-    await ButtonPageObject.clickComponent();
-    await expect(await ButtonPageObject.didOnClickCallbackFire()).toBeTruthy();
+    await ButtonPageObject.click(ButtonPageObject._primaryComponent);
+    await expect(await ButtonPageObject.waitForOnClickCallbackToFire()).toBeTruthy(
+      'Mouse click failed to fire the onClick callback for the primary button.',
+    );
+
     await expect(await ButtonPageObject.didAssertPopup()).toBeFalsy(ButtonPageObject.ERRORMESSAGE_ASSERT);
 
     await ButtonPageObject.clickComponent(); // Reset Button State
   });
 
   it('Validate OnClick() callback was fired -> Type "Enter"', async () => {
-    await ButtonPageObject.sendKey(ButtonSelector.PrimaryButton, Keys.ENTER);
-    await expect(await ButtonPageObject.didOnClickCallbackFire()).toBeTruthy();
+    await ButtonPageObject.sendKeys(ButtonPageObject._primaryComponent, [Keys.ENTER]);
+    await expect(await ButtonPageObject.waitForOnClickCallbackToFire()).toBeTruthy(
+      "'Enter' keypress failed to fire the onClick callback for the primary button.",
```

Comment:
Same as above:

"Pressing the "Enter" key on the primary button failed to fire the onClick() callback"